packages/api/src/command/medical/patient/update-patient.ts

```diff
@@ -5,7 +5,8 @@ import { getPatientOrFail } from "./get-patient";
 import { sanitize, validate } from "./shared";
 
 type PatientNoExternalData = Omit<PatientData, "externalData">;
-export type PatientUpdateCmd = BaseUpdateCmdWithCustomer & PatientNoExternalData;
+export type PatientUpdateCmd = BaseUpdateCmdWithCustomer &
+  PatientNoExternalData & { externalId?: string };
```

Comment:
Do we have a use case for them to update an externalId of the patient?

Reply:
They set the wrong id and want to update it